Assets/Scripts/DrawDetector.cs

```diff
@@ -1 +1,20 @@
+using UnityEngine;
 
+public class DrawDetector : MonoBehaviour
+{
+    [SerializeField] private GameCalculator _gameCalculator;
+
+    [SerializeField] private TextOutput _textOutput;
+
+    [SerializeField] private MoveCounter _moveCounter;
+
+    public const int NumberOfMove = 9;
```

Comment:
MaxNumberOfMoves
Also i dont think you need this const because you should check if there is any empty space on the field rather than hardcoding max moves const.